dali/python/nvidia/dali/ops.py

```diff
@@ -314,14 +317,19 @@ def __init__(self, inputs, op, **kwargs):
         # Argument inputs
         for k in sorted(kwargs.keys()):
             if k not in ["name"]:
-                if not isinstance(kwargs[k], _EdgeReference):
+                arg_inp = kwargs[k]
+                if arg_inp is None:
+                    continue
+                if type(arg_inp) is _Constant:
```

Comment:
You could write this for consistency:
```suggestion
                if isinstance(arg_inp, _Constant):
```

Reply:
ok